types.go

```diff
@@ -1,11 +1,14 @@
 package cnc
 
-import "github.com/gogo/protobuf/types"
+import (
+	"github.com/gogo/protobuf/types"
+)
 
 // SubmitPFDRequest represents a request to submit a PayForData transaction.
 type SubmitPFDRequest struct {
 	NamespaceID string `json:"namespace_id"`
 	Data        string `json:"data"`
+	Fee         int64  `json:"fee"`
```

Comment:
What is the expected range of values? With decimals in mind, would `int64` be sufficient? `GasPrice` is generally handled as big integer.

Reply:
Again, we my want to change this later, and we need would need to fix this also in celestia-node (see: previous comment).

Reply:
Fee has the same type as the `amount` field in other requests, as it ends up as a cosmos `math.Int`. Yes we might want to change this later

The `GasLimit` may also change to an int64 soon:
https://github.com/celestiaorg/celestia-app/issues/1133

But will keep go-cnc in mind when these changes occur!